services/userService.js

```diff
@@ -0,0 +1,27 @@
+const userDao = require('../models/userDao');
+const { baseError } = require('../utils/error');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+
+const getUserByAccount = async (account) => {
+  return await userDao.getUserByAccount(account);
+};
+
+const signIn = async (account, password) => {
+  const user = await userDao.getUserByAccount(account);
+  if (!user) {
+    throw new baseError('USER_NOT_VALID');
+  }
+
+  const checkHash = await bcrypt.compare(password, user.password);
+
+  if (!checkHash) {
+    throw new baseError('PASSWORD_NOT_MATCH', 401);
+  }
+  const payLoad = { account: account, id: user.id };
```

Comment:
로그인 한 계정을 굳이 payload에 담을 필요는 없을 것 같습니다.
```suggestion
  const payLoad = { id: user.id };
```